src/sponge/mod.rs

```diff
@@ -4,6 +4,8 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-pub mod sponge;
+mod sponge;
 
+#[cfg(feature = "std")]
```

Comment:
Why is `std` instead of `alloc` the feature for this fn?

Reply:
This is related to the plonk requirements. Plonk is defined under `std` while `Vec` is defined under `alloc`

Reply:
We need to work on that at some point. Port plonk to no_std using alloc and vec. Otherways this is non-sensical...

Reply:
Cool, we can register an issue to address this later on